get-config.js

```diff
@@ -1,6 +1,6 @@
 
 module.exports = function getConfig() {
-    const configPath = process.cwd() + '/src/config';
+    const configPath = process.cwd() + '/src/config.js';
```

Comment:
hmm, why did you have to change this?

Reply:
Wasn't finding the config.js file locally. Worked when I changed it. 🤷‍♀